src/js/state/Appearance/index.ts

```diff
@@ -1,17 +1,24 @@
 import {createSlice, PayloadAction} from "@reduxjs/toolkit"
 import {SectionData} from "src/pkg/sectional"
 import {State} from "../types"
+import {HistoryView, QueriesView, SectionName} from "./types"
 
 const init = () => ({
   sidebarIsOpen: true,
   sidebarWidth: 230,
-  sidebarSections: [{id: "pools"}, {id: "queries"}, {id: "history"}]
+  sidebarSections: [{id: "pools"}, {id: "queries"}, {id: "history"}],
```

Comment:
This was a bad idea on my part. I think this should just be hard coded in the jsx.

Reply:
Yeah the new sidebar doesn't use those, I'll remove when we remove the ff